packages/docs-app/src/examples/select-examples/multiSelectExample.tsx

```diff
@@ -125,13 +139,27 @@ export class MultiSelectExample extends React.PureComponent<ExampleProps, MultiS
                     onItemSelect={this.handleFilmSelect}
                     onItemsPaste={this.handleFilmsPaste}
                     popoverProps={{ matchTargetWidth, minimal: popoverMinimal }}
+                    // Without matchTargetWidth there is no width defined in any of TagInput's
+                    // grandparents when it's rendered through usePortal, so it will never flex-wrap
+                    // and infinitely grow horizontally. To address this, if there is no width guidance
+                    // from matchTargetWidth, explicitly set a width to so Tags will flex-wrap.
```

Comment:
Reason that Default TagInput (not customTarget) doesn't need this is because the TagInput is not rendered within the Popover so it's within the regular DOM hierarchy, meaning it can look to it's ancestors to eventually grab a defined width somewhere to inherit

Reply:
Instead of adding this to the docs let's add it into the underlying component. thinking is that _everyone_ will need to do something like this, so we may as well provide a reasonable default.